### ISSUE TITLE
[EDA] eda-ui is deployment fail

### ISSUE THREAD

**Issue by ksaegusa** [ansible/ansible-ui#951]
I am building with eda-operator and eda-ui is giving me an error.
When I deployed it days ago, it was fine.  

unknown `eda_server` variable

I've investigated and I'm getting an error starting at 2.4.455.

## pod status
```sh
# kubectl get pod -n eda
NAME                                                      READY   STATUS             RESTARTS       AGE
eda-server-operator-controller-manager-858495bdbf-j2zzb   2/2     Running            12 (86m ago)   6d
eda-redis-8554b848f9-ltldp                                1/1     Running            0              17m
eda-postgres-13-0                                         1/1     Running            0              17m
eda-default-worker-758d5dd588-g4cv8                       1/1     Running            0              16m
eda-default-worker-758d5dd588-bcdj2                       1/1     Running            0              16m
eda-activation-worker-5748b7f98c-cr8pz                    1/1     Running            0              16m
eda-activation-worker-5748b7f98c-xjftr                    1/1     Running            0              16m
eda-activation-worker-5748b7f98c-lf294                    1/1     Running            0              16m
eda-activation-worker-5748b7f98c-tpsdt                    1/1     Running            0              16m
eda-activation-worker-5748b7f98c-k8wtt                    1/1     Running            0              16m
eda-scheduler-86b874d978-vhjbz                            1/1     Running            0              16m
eda-api-678bfcfdf7-sdgv6                                  2/2     Running            0              17m
eda-ui-5bc56f5d79-9ggqj                                   0/1     CrashLoopBackOff   8 (60s ago)    17m
```

## Error Message
```sh
# kubectl logs eda-ui-5bc56f5d79-9ggqj -n eda
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
/docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
20-envsubst-on-templates.sh: Running envsubst on /etc/nginx/templates/default.conf.template to /etc/nginx/conf.d/default.conf
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
2023/09/26 09:57:47 [emerg] 1#1: unknown "eda_server" variable
nginx: [emerg] unknown "eda_server" variable
```

## reference information
```
  # image_web_version: 2.4.469 Error
  # image_web_version: 2.4.467 Error
  # image_web_version: 2.4.460 Error
  # image_web_version: 2.4.455 Error
  # image_web_version: 2.4.454 OK
  # image_web_version: 2.4.453 OK
  # image_web_version: 2.4.452 OK
  # image_web_version: 2.4.450 OK
```

**Comment by kurokobo** [comment]
This was already addressed in ansible/eda-server-operator#133, but the latest operator has another issue for eda-ui: https://github.com/ansible/eda-server-operator/issues/134

So, currently there is no way to deploy EDA Server with UI by EDA Server Operator without pinning `eda-ui` with old tag by `image_web_version`.

Also, in my opinion, if the changes in `ansible-ui` causes a deployment failure, the first choice would be to address it in the Operator-side. Therefore, I think it is better that this kind of issue be reported in the Operator repository.

**Comment by ksaegusa** [comment]
Sorry to dig up a problem that has already been resolved.

Thanks for the advice on where to report it and also for the workaround.
I will wait for the related issue to be resolved.
